app/Console.php

```diff
@@ -94,6 +94,18 @@ class Console extends Model
             'provider' => 'Digital Ocean',
             'route' => 'digital-ocean',
         ],
+        [
+            'name' => 'Apple Developer Program',
+            'url' => 'https://developer.apple.com/account',
+            'provider' => 'Apple',
+            'route' => 'developer',
```

Comment:
Haha, sorry, I meant: `apple/developer` and `apple/connect`. Should've been more clear!

Reply:
Lol, I almost did that too. Figured you were being explicit rather than implicit 😁. No worries, I'll update it tonight.